coalib/results/SourceRange.py

```diff
@@ -62,12 +69,23 @@ def from_position(cls, filename, text, start_position, end_position):
         start_line, start_column = cls.calc_line_col(text, start_position)
         end_line, end_column = cls.calc_line_col(text, end_position)
         return cls(SourcePosition(filename, start_line, start_column),
-                   SourcePosition(filename, end_line, end_column))
+                   SourcePosition(filename, end_line, end_column),
+                   text=text,
+                   start_position=start_position,
+                   end_position=end_position)
 
     @property
     def file(self):
         return self.start.file
 
+    @property
+    def position(self):
```

Comment:
But, if I do from_values with text, start and end You should calculate start_position, end_position here